defos/src/defos_mac.mm

```diff
@@ -93,6 +93,20 @@ void defos_set_window_title(const char* title_lua) {
     [window setTitle:title];
 }
 
+void defos_set_window_icon(const char *icon_path)
+{
+    NSString *path = [NSString stringWithUTF8String:icon_path];
```

Comment:
This string gets created and autoreleased, meaning it will get garbage collected by the nearest autorelease pool. We don't know how Defold works internally, so we don't know if it has an autorelease pool in all contexts this function might get called. It would be safest to surround this with an `@autorelease { ... }`.

Reply:
ok, i'll do this. 
I think Defold has no autorelease pool in all context, i found this issue with other my NE :
https://forum.defold.com/t/native-extensions/4946/149?u=agulev

Reply:
done